include/bluetooth/conn.h

```diff
@@ -323,6 +327,35 @@ int bt_conn_security(struct bt_conn *conn, bt_security_t sec);
  */
 u8_t bt_conn_enc_key_size(struct bt_conn *conn);
 
+enum bt_security_err {
```

Comment:
I would rename this to `bt_security_status` and use `BT_SECURITY_STATUS_*`, because `ERR_SUCCESS` makes no sense